src/rand.rs

```diff
@@ -100,14 +100,14 @@ impl sealed::Sealed for SystemRandom {}
         target_os = "macos",
         target_os = "ios",
         target_os = "fuchsia",
-        windows
+        target_os = "windows"
```

Comment:
How should `fill_impl` be implemented for UEFI targets? Please provide an implementation.